assembl/static2/js/app/components/debate/common/topPostFormContainer.jsx

```diff
@@ -82,10 +83,12 @@ class TopPostFormContainer extends React.Component<*, TopPostFormContainerProps,
   }
 
   render() {
-    const { ideaId, refetchIdea, messageColumns = [], isColumnViewInline } = this.props;
+    const { ideaId, refetchIdea, messageColumns = [], isColumnViewInline, coarsePostsCount } = this.props;
     const columnsInfos = this.getColumnsInfos();
+    const { sticky } = this.state;
+    const containerClassNames = sticky && messageColumns.length <= 1 && coarsePostsCount >= 1 ? 'top-post-sticky' : '';
```

Comment:
Can leverage `classnames` here

Reply:
Isn't classnames only useful if you have a case with certain classes and another with the same + others ? In this case there is only one class which is present under certain conditions so I don't see why the library is necessary.

Reply:
I agree, classnames is overkill just for one class.